data_wrangling/missing.ipynb

```diff
@@ -0,0 +1,997 @@
+{
```

Comment:
<p>Is this in percentage? Please indicate</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Data-Epic/gbotemi_bolarinwa/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>